modules/tinymce/src/core/main/ts/dom/NodeType.ts

```diff
@@ -59,7 +59,7 @@ const hasPropValue = function (propName: string, propValue: any) {
   };
 };
 
-const hasAttribute = function (attrName: string, attrValue?: string) {
+const hasAttribute = function (attrName: string, _attrValue?: string) {
```

Comment:
Should we just remove this attribute instead?

Reply:
Will do.

Reply:
Done